.github/workflows/ci.yml

```diff
@@ -10,4 +10,4 @@ jobs:
       - name: Checkout the code
         uses: actions/checkout@v2
       - name: Build schema from source
-        run: docker-compose -f docker-compose-ci.yml run build_from_source
+        run: docker-compose -f docker-compose-ci.yml run update_from_remote_schema
```

Comment:
nitpick: I'm not sure if this improvement belongs in this particular PR

Reply:
It belongs indeed. 
This makes sure client.ts is regenerated for PR builds.